src/argus_htmx/incidents/views.py

```diff
@@ -135,11 +140,15 @@ def get_form_data(request, formclass: forms.Form):
 
 @require_POST
 def incidents_update(request: HtmxHttpRequest):
-    form_id = request.htmx.trigger_name or request.htmx.trigger
-    formclass, queryset = INCIDENT_UPDATE_FORM_NAMES.get(form_id, (None, None))
+    form_name = request.POST.get("form_name")
```

Comment:
This is still ugly.

Reply:
can't speak for @hmpf but I think the parameter name "form_name" is a bit confusing for a consumer (not a regular user, but someone who either programs against argus or looks at the devtools). They just want "do something" with the incidents. Whether or not there are forms associated with the implementation is irrelevant. Consider the following two options:
```
POST /incidents/update?form_name=close&selected_incidents=1&selected_incidents=2
```
vs
```
POST /incidents/update?action=close&incident_id=1&incident_id=2
```

which of the two looks more "natural"? I'd say the second one. Which is why I (still) recommend using the latter names instead of the former.

Btw, it is then totally ok to go:
```python
form_name  = request.POST.get("action")
incidents = request.POST.getlist("incident_id")
```

I concede that it's not a rest api, so this may not be as high a concern, but I think it still has value to make requests urls/params/bodies look nice and clean

Reply:
```POST /incidents/update/close/?incident_id=1&incident_id=2```

is my preference.

With

```
def incidents_update(request: HtmxHttpRequest, action: str):
```

and 

```
path("update/<str:action>/", views.incidents_update, name="incidents-update"),
```

It's something other django developers will more intuitively understand.

Reply:
Not that we do it my way *now* but I'm extremely likely to rewrite it like that.

Reply:
that's a nice solution as well :)